src/libraries/System.IO.FileSystem/src/System/IO/FileSystem.Windows.cs

```diff
@@ -369,42 +369,44 @@ public static void SetAttributes(string fullPath, FileAttributes attributes)
             }
         }
 
-        public static void SetCreationTime(string fullPath, DateTimeOffset time, bool asDirectory)
+        // Default values indicate "no change".  Use defaults so that we don't force callsites to be aware of the default values
+        private static unsafe void SetFileTime(
+            string fullPath,
+            bool asDirectory,
+            long creationTime = -1,
```

Comment:
I couldn't find documentation of this "-1" behavior.

Reply:
It is what the existing did. I just moved it around.

I am not able the documentation of this behavior either.

Reply:
`SetFileInformationByHandle` doesn't mention it but you can find this little detail mentioned in `SetFileTime`'s documentation: https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-setfiletime.

Reply:
Oh, good!